.github/workflows/unittests.yml

```diff
@@ -100,11 +100,13 @@ jobs:
     if: "!contains(github.event.head_commit.message, 'ci-skip')"
 
     steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-python@v1
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
What does fetch-depth do?

Reply:
It fetches the whole history including the tags, which we need because the script call `git describe --tags`